### PR TITLE
migrated old commands to discordjs

### PR DESCRIPTION
**Pull request by tikohov20**
Pls approve

### CODE REVIEW

File: Messages/index.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        const janaxper = [
+          'ջան ախպեր',
+          'ոնց ախպեր',
+          'բա ախպեր'
+        ];
+        await message.reply(janaxper[Math.floor(Math.random()*janaxper.length)])
+      }
+    ],
+    [
+      ['միբան','միբան', 'miban', 'miban', 'jackbox', 'ջեքբոքս', 'jackbox', 'ջեքբոքս', 'cs'],
```
Comment (karapnaran) [inline Messages/index.js]: unacceptable, pls remove half

**Review by karapnaran — CHANGES_REQUESTED** [review]

File: Messages/index.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+          'էն օրը '
+        ];
+        const tegher = [
+          'տանը ',
+          'ջիմում '
+        ];
+        const text = orer[Math.floor(Math.random()*orer.length)]+
+          tegher[Math.floor(Math.random()*tegher.length)]+
+          'կշռվեցի '+
+          (Math.floor(Math.random() * (105 - 79) + 79)).toString() + ' կիլո էի';
```
Comment (karapnaran) [inline Messages/index.js:28]: @tikohov20 please update the numbers, thank you in advance 
80fe9a85-d09f-415d-8fce-597b70256d7b

File: index.js [hunk 3]
```diff
@@ -24,32 +25,30 @@ client.once(Events.ClientReady, c => {
...
+        // }
     })
 
     // handle commands
     c.on(Events.InteractionCreate, async interaction => {
         if (!interaction.isChatInputCommand()) return;
 
         if (interaction.commandName === 'pupulachap') {
-            console.log(interaction);
+            // const userId = 894153328353759243;
```
Comment (karapnaran) [inline index.js]: delet this

File: Messages/index.js (hunk 2 above)
Comment (HrachAsatryan) [inline Messages/index.js:28]: Numbers are up to date.

File: Messages/index.js (hunk 1 above)
Comment (HrachAsatryan) [reply Messages/index.js]: add ցս, fortnite, ֆորթնայթ

**Review by HrachAsatryan — APPROVED** [review]

File: Messages/index.js [hunk 4]
```diff
@@ -114,6 +114,24 @@ const buildSimpleHandlers = () => {
           await message.react('🤬')
         ])
       }
+    ],
+    [
+      (text) => text === '/tusbomb',
+      async (message) => {
+        const tus = [
```
Comment (HrachAsatryan) [inline Messages/index.js:121]: change to read 5 random messages from #Tus Moments Museum

**Review by HrachAsatryan — CHANGES_REQUESTED** [review]

File: Messages/index.js (hunk 4 above)
Comment (tikohov20) [reply Messages/index.js:121]: sooooon :tm:

**Pull request merged**